pkg/datastore/datastore.go

```diff
@@ -83,8 +83,7 @@ type Helper interface {
 }
 
 type TestHelper interface {
-	DropTables(records ...Record) error                       // Drop DB tables by records
-	Truncate(tableNames ...string) error                      // Truncates DB tables
-	TruncateCascade(cascade bool, tableNames ...string) error // Truncates DB tables, with an option to truncate them in a cascading fashion
-	HasTable(tableName string) (bool, error)                  // Checks if DB table exists
+	DropTables(records ...Record) error      // Drop DB tables by records
+	Truncate(records ...Record) error        // Truncates DB tables
+	HasTable(tableName string) (bool, error) // Checks if DB table exists
```

Comment:
@jeyhun please be consistent with interface either we support both tableName and records or just records. I don't want the reactors keep pouring in. You could the following interface if it makes sense

DropTables(records ...Record)
DropTablesByName(names ...string)

Please stick to this signature if you are refactoring. Note this would have repercussions once consumers upgrade the library. So it would be better to finalize and make change, else just leave it to callers.